Could not find a declaration file for module '@reactivex/rxjs' (5.5.x)

**RxJS version:**
5.5.0, 5.5.1, 5.5.2

**Code to reproduce:**
*index.ts*
```typescript
import { Subject } from "@reactivex/rxjs";

const test = new Subject<boolean>();

test.next(true);
```

*package.json*
```json
{
  "dependencies": {
    "@reactivex/rxjs": "5.5.0",
    "typescript": "^2.5.3"
  }
}

*tsconfig.json*
```json
{
    "compilerOptions": {
        "noImplicitAny": true
    }
}
```

**Expected behavior:**
No errors.

**Actual behavior:**
> index.ts(1,25): error TS7016: Could not find a declaration file for module '@reactivex/rxjs'. 'C:/Users/mattd/Source/Examples/rxjs-bug/node_modules/@reactivex/rxjs/index.js' implicitly has an 'any' type.
>  Try `npm install @types/@reactivex/rxjs` if it exists or add a new declaration (.d.ts) file containing `declare module '@reactivex/rxjs';`

**Additional information:**
The `package.json` of rxjs contains `"typings": "./dist/package/typings/Rx.d.ts"`, but there is no `typings` folder in the `package` directory. I expect that it should be changed to `"typings": "./dist/typings/Rx.d.ts"`, because that appears to work.

--- a/package.json
+++ b/package.json
@@ -236,7 +236,7 @@
   "engines": {
     "npm": ">=2.0.0"
   },
-  "typings": "./dist/package/typings/Rx.d.ts",
+  "typings": "./dist/package/Rx.d.ts",
   "dependencies": {
     "symbol-observable": "^1.0.1"
   }

